common/static/common/js/spec/components/paging_collection.js

```diff
@@ -32,6 +32,44 @@ define(["backbone.paginator", "backbone"], function(BackbonePaginator, Backbone)
             this.currentPage = currentPage;
             this.start = start;
             return response.items;
+        },
+
+        setPage: function (page) {
```

Comment:
I'm confused by this method. Why do you have to reimplement the method that is already on the paging collection? Wouldn't it be better to test using the collection's own method.